DOC-4707 - Change the wording around crlCheckSoftFail setting to indicate known limitations (#1099)
corda/corda-docs-portal: content/en/platform/corda/1.2/cenm/identity-manager.md

configuration may need to be the private IP address, whilst external nodes would use the machines external
IP/DNS name to connect to Identity Manager.

> If using a fixed IP instead of a DNS address, this can subsequently prevent CRL endpoints in your CENM services' certificates from being reachable if you decide to move CENM services to a new cluster. R3 will not support issues created as a result of fixed IP addresses being used in CRL endpoints. R3 recommend to use only DNS addresses.

Reviewer: ```suggestion
Using a fixed IP address instead of a DNS address can prevent CRL endpoints in your CENM services' certificates from being reachable if you subsequently move CENM services to a new cluster. R3 do not support issues created as a result of fixed IP addresses being used in CRL endpoints. R3 recommends using only DNS addresses.
```

Author: HI Nadine, updated all files .
